services/minespace-web/src/styles/overrides/antd-overrides.scss

```diff
@@ -23,10 +23,14 @@
 .ant-layout-footer {
   z-index: 8;
   width: 100%;
-  min-height: 80px;
+  min-height: $layout-footer-height;
   bottom: 0;
   padding: 0;
   height: $layout-footer-height;
+
+  @include screen-lg {
+    height: unset;
```

Comment:
footer bug!
Any screen size smaller than lg (somewhere ~1000px) on MS, you lose footer items and it appears to "float". It does this at the breakpoint where the items change from a horizontal list to vertical and so the height _should_ change. This is actually a pretty reasonable screen width to use with MS so I dealt with it.